### PR TITLE
[new release] reshowcase (5.3.0)

### PR DESCRIPTION
**Pull request by jchavarri**
A tool to create demos for your ReasonReact components

- Project page: <a href="https://github.com/ahrefs/reshowcase">https://github.com/ahrefs/reshowcase</a>
- Documentation: <a href="https://github.com/ahrefs/reshowcase">https://github.com/ahrefs/reshowcase</a>

##### CHANGES:

Migrate to Melange v2.

### CODE REVIEW

File: packages/reshowcase/reshowcase.5.3.0/opam [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+doc: "https://github.com/ahrefs/reshowcase"
+bug-reports: "https://github.com/ahrefs/reshowcase/issues"
+depends: [
+  "dune" {>= "3.8"}
+  "ocaml"
+  "melange" {>= "2.0.0"}
+  "reason" {>= "3.10.0"}
+  "reason-react"
+  "reason-react-ppx"
+  "ocaml-lsp-server" {with-test}
```
Comment (haochenx) [inline packages/reshowcase/reshowcase.5.3.0/opam:17]: Maybe for this one `with-dev-setup` is intended instead?
```suggestion
  "ocaml-lsp-server" {with-dev-setup}
```

Comment (jchavarri) [reply packages/reshowcase/reshowcase.5.3.0/opam]: sounds good (looking forward to 2.2 😄 )

Comment (haochenx) [reply packages/reshowcase/reshowcase.5.3.0/opam]: (I figured out that this package likely not requiring `ocaml-lsp-server` to build / test and installing it when not necessary is actually a little wasteful)

**Review by haochenx — APPROVED** [review]
All looks good to me. Thanks!

**Comment by avsm** [comment]
Thanks! You may want to announce this on <https://discuss.ocaml.org>, where we have a Community category and an `announce` tag for this purpose.

**Pull request merged**